tests/playwright/src/model/workbench/status-bar.ts

```diff
@@ -0,0 +1,54 @@
+/**********************************************************************
+ * Copyright (C) 2023-2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Locator, Page } from '@playwright/test';
+import { expect } from '@playwright/test';
+
+export class StatusBar {
+  readonly page: Page;
+  readonly kindInstallationButton: Locator;
+  readonly kindDialogWindow: Locator;
+  readonly dialogYesButton: Locator;
+  readonly dialogCancelButton: Locator;
+  readonly dialogOkButton: Locator;
+  readonly dialogCloseButton: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.kindInstallationButton = this.page.getByTitle(
+      'Kind not found on your system, click to download and install it',
+    );
+    this.kindDialogWindow = this.page.getByRole('dialog', { name: 'Kind' });
+    this.dialogYesButton = this.kindDialogWindow.getByRole('button', { name: 'Yes' });
+    this.dialogCancelButton = this.kindDialogWindow.getByRole('button', { name: 'Cancel' });
+    this.dialogOkButton = this.kindDialogWindow.getByRole('button', { name: 'OK' });
+    this.dialogCloseButton = this.kindDialogWindow.getByRole('button', { name: 'Close' });
+  }
+
+  public async installKindCLI(): Promise<void> {
+    await this.kindInstallationButton.click({ timeout: 5000 });
+    await expect(this.kindDialogWindow).toBeVisible();
+    await this.dialogYesButton.click({ timeout: 5000 });
+    await this.dialogYesButton.click({ timeout: 5000 });
+    await this.dialogCloseButton.click({ timeout: 5000 });
+  }
+
+  public getKindInstallationButton(): Locator {
+    return this.kindInstallationButton;
```

Comment:
What is the point of this method? We can directly access the locator field that it returns if it's specifically needed.